atom-publisher-lib/build.sbt

```diff
@@ -18,7 +18,14 @@ startDynamoDBLocal <<= startDynamoDBLocal.dependsOn(compile in Test)
 test in Test <<= (test in Test).dependsOn(startDynamoDBLocal)
 testOptions in Test <+= dynamoDBLocalTestCleanup
 
+dependencyOverrides += "org.apache.thrift" % "libthrift" % "0.9.1"
```

Comment:
Which library is forcing us to override? Ideally we could just upgrade it to avoid this